Recipely/Recipely/ProfileScreen/View/ProfileViewController.swift

```diff
@@ -0,0 +1,203 @@
+        case .info:
+            guard
+                let cell = tableView
+                .dequeueReusableCell(withIdentifier: AppConstants.infoIdentifier) as? InfoTableViewCell,
+                let model = cells[indexPath.row] as? Info
+            else {
+                return UITableViewCell()
+            }
+            cell.configureCell(model: model, tapButton: onTapHandler)
+            return cell
+        case .bonuses:
+            guard
+                let cell = tableView
+                .dequeueReusableCell(withIdentifier: AppConstants.bonusesIdentifier) as? BonusesTableViewCell,
+                let model = cells[indexPath.row] as? Bonuses
+            else {
+                return UITableViewCell()
+            }
+            cell.configureCell(model: model, tapButton: arrowTapHandler)
+            return cell
+        case .terms:
+            guard
+                let cell = tableView
+                .dequeueReusableCell(withIdentifier: AppConstants.termsIdentifier) as? TermsTableViewCell,
+                let model = cells[indexPath.row] as? Terms
+            else {
+                return UITableViewCell()
+            }
+            cell.configureCell(model: model)
+            return cell
+        case .logOut:
+            guard
+                let cell = tableView
+                .dequeueReusableCell(withIdentifier: AppConstants.logOutIdentifier) as? LogOutTableViewCell,
+                let model = cells[indexPath.row] as? LogOut
+            else {
+                return UITableViewCell()
+            }
+            cell.configureCell(model: model)
+            return cell
+        }
+    }
+}
+
+// MARK: - Extension ProfileViewController + ProfileViewProtocol
+
+/// Имплементация ProfileViewProtocol
+extension ProfileViewController: ProfileViewProtocol {
+    // настройка алерта
+    func configureAlert() {
+        let alertController = UIAlertController(
+            title: AppConstants.changeAlertTitle,
+            message: nil,
+            preferredStyle: .alert
+        )
+        let actionCancel = UIAlertAction(title: AppConstants.cancel, style: .default)
+        let actionOk = UIAlertAction(title: AppConstants.ok, style: .cancel) { _ in
+            guard let text = alertController.textFields?.first?.text else { return }
+            self.tapActionOk(text: text)
+        }
+        alertController.addTextField { title in
+            title.placeholder = AppConstants.placeholderText
+        }
+        alertController.addAction(actionCancel)
+        alertController.addAction(actionOk)
+        present(alertController, animated: true)
+    }
+
+    /// Уведомляем презентер о нажатии на "ок" в алерте
+    /// - Parametr: текст из текстфилда
+    func tapActionOk(text: String) {
+        profilePresenter?.changeName(text: text)
+    }
+
+    /// Изменение текста лейбла с именем
+    /// - Parametr: новое имя
+    func changeLabel(updateName: String) {
+        cells[0] = Info(
```

Comment:
```suggestion
        cells[0] = Info(
```
[0] не безопасный код